lib/quadquizaminos_web/views/layout_view.ex

```diff
@@ -1,3 +1,17 @@
 defmodule QuadquizaminosWeb.LayoutView do
   use QuadquizaminosWeb, :view
+
+  defp admin_dropdown do
+    ~E"""
```

Comment:
You can add a dropdown caret to indicate its a dropdown
` <i class="fa fa-caret-down"></i>`

Reply:
okay